Air Conditioning/Daikin RTD-NET FBA-A(9).xml

```diff
@@ -0,0 +1,239 @@
+        <SlaveId>$[SlaveId]</SlaveId>
+        <MaxPrefetchGroupSize>100</MaxPrefetchGroupSize>
+        <IpAddress>$[IpAddress]</IpAddress>
+      </DeviceProperties>
+      <Devices>
+        <Device>
+          <Name>Fanspeed</Name>
+          <Model>ModbusMultiValueSwitch</Model>
+          <Id>-2</Id>
+          <DeviceProperties>
+            <DeviceType>6011</DeviceType>
+            <InternalPollInterval>2500</InternalPollInterval>
+            <InitializeScript></InitializeScript>
+            <ReadScript></ReadScript>
+            <PrefetchModeId>0</PrefetchModeId>
+            <ServiceAttributesFormulas>[{"AttributeDefinition":{"Name":"Damper Control"},"ReadFormula":"Switch(MODBUSR(H, 30, Uint16), 0, \u0022Auto\u0022, 1, \u0022Cross Flow / Heat Recovery\u0022, 2, \u0022Bypass\u0022, \u0022Unknown\u0022);"},{"AttributeDefinition":{"Name":"VAM Fanspeed"},"ReadFormula":"Switch(MODBUSR(H, 31, Uint16),1, \u0022Low\u0022, 2, \u0022High\u0022, \u0022Unknown\u0022);"}]</ServiceAttributesFormulas>
+            <ServiceActionsScripts></ServiceActionsScripts>
+            <CustomDeviceVariables />
+            <Value0Index>64</Value0Index>
+            <Value1Index>1</Value1Index>
+            <Value2Index>2</Value2Index>
+            <Value3Index>3</Value3Index>
+            <Value4Index>68</Value4Index>
+            <Value5Index>69</Value5Index>
+            <Value6Index>70</Value6Index>
+            <Value7Index>71</Value7Index>
+            <Value8Index>72</Value8Index>
+            <Value9Index>73</Value9Index>
+            <Value0Name>${level_0Of3}</Value0Name>
+            <Value1Name>Low</Value1Name>
+            <Value2Name>Medium</Value2Name>
+            <Value3Name>High</Value3Name>
+            <Value4Name></Value4Name>
+            <Value5Name></Value5Name>
+            <Value6Name></Value6Name>
+            <Value7Name></Value7Name>
+            <Value8Name></Value8Name>
+            <Value9Name></Value9Name>
+            <Value0IconId>20</Value0IconId>
+            <Value1IconId>21</Value1IconId>
+            <Value2IconId>22</Value2IconId>
+            <Value3IconId>23</Value3IconId>
+            <Value4IconId>0</Value4IconId>
+            <Value5IconId>0</Value5IconId>
+            <Value6IconId>0</Value6IconId>
+            <Value7IconId>0</Value7IconId>
+            <Value8IconId>0</Value8IconId>
+            <Value9IconId>0</Value9IconId>
+            <OffStateValue>-1</OffStateValue>
```

Comment:
Vazne ten ventilator nema vypnuty stav ?

Reply:
Má ale samostatne. Že môžeš zapnúť vypnúť a ostane tam rovnaká úroveň ventilácie. Môžem to teoreticky spraviť do jedného cez nejakú funkciu ak treba

Reply:
normalne je tam presne na to ten vypnuty stav, ale v modbuse to asi neni dorobene, tak to nechame tak